pkg/pillar/cmd/zedagent/handlelocation.go

```diff
@@ -59,28 +60,11 @@ func locationTimerTask(ctx *zedagentContext, handleChannel chan interface{}) {
 	for {
 		select {
 		case <-cloudTicker.C:
-			locInfo := getLocationInfo(ctx)
-			if locInfo == nil {
-				// Not available.
-				break
-			}
-			start := time.Now()
-			cloudIteration++
-			publishLocationToController(locInfo, cloudIteration)
-			ctx.ps.CheckMaxTimeTopic(wdName, "publishLocationToController", start,
-				warningTime, errorTime)
-
+			publishLocation(ctx, &cloudIteration, wdName, ControllerDest)
```

Comment:
@milan-zededa don't we want to publish periodically to AllDest here? (and, if so, rename the tcloudTicker variable)?

Reply:
With the current code in the PR we only publish location to the LOC when we publish all info which is when the controller epoch changes.

Reply:
But see below - I might have missed a new use of triggerPublishAllInfo(). Makes sense to add comments about that new usage here resulting in publishing the location and check that the comment for triggerPublishAllInfo isn't stating something misleading abouts its usage.

Reply:
No sure I understand comments right. The default behavior is unchanged. If not - this is a bug.